include/sparrow/layout/null_array.hpp

```diff
@@ -183,12 +199,63 @@ namespace sparrow
      * null_array implementation *
      *****************************/
 
+    inline null_array::null_array(
+        size_t length,
+        std::optional<std::string_view>&& name,
+        std::optional<std::string_view>&& metadata
+    )
+        : m_proxy(create_proxy(length, std::move(name), std::move(metadata)))
+    {
+    }
+
+    inline arrow_proxy null_array::create_proxy(
+        size_t length,
+        std::optional<std::string_view>&& name,
+        std::optional<std::string_view>&& metadata
+    )
+    {
+        using namespace std::literals;
+        ArrowSchema schema = make_arrow_schema(
+            "n"sv,
+            std::forward<std::optional<std::string_view>>(name),
+            std::forward<std::optional<std::string_view>>(metadata),
```

Comment:
It's not a deducing context, so no need for `std::forward` here. Prefer moving the arguments instead (or passing them by value if you need them later in the function).

Reply:
done